sdk/mapper.js

```diff
@@ -66,8 +66,12 @@ class Mapper {
         }
       }
     };
-    if (responseApi.hasReturn()) {
-      payload[m.command_reply][m.result][m.response_return] = responseApi.getReturnType();
+    try {
+      if (responseApi.hasReturn()) {
+        payload[m.command_reply][m.result][m.response_return] = responseApi.getReturnType();
```

Comment:
`hasReturn()` depends on `getServiceSchema()`. If there's no schema it throws an error. If there's an error always on this point, then the CLI won't work for middlewares that rely on `newResponse()`.

Also, middlewares won't work even in `start` mode until #15 gets fixed.